CMakeLists.txt

```diff
@@ -99,9 +99,8 @@ set(CMAKE_CXX_STANDARD 20)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
-add_compile_options(-Wno-literal-suffix)
 if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
-    add_compile_options(-fconcepts)
+    add_compile_options(-Wno-literal-suffix)
```

Comment:
Why do you need to change the GNU compiler ID to make Clang work? I'd think that it would only matter with the Clang compiler ID below ^^'

Reply:
Nope, `CMAKE_CXX_COMPILER_ID` picks up the host compiler, which is GCC for me. We do not clear the compile options when switching compilers, so this would present itself as an unknown command-line option on Clang later on.

Of course that should be fixed too, but that's another round of CMake magic.